yourgov/components/FormMobileFoodVendorPermit/steps/FormStep10Review.tsx

```diff
@@ -55,8 +54,8 @@ export function FormStep10Review({ headingsLevel }: FormReviewProps) {
 	return (
 		<Stack gap={3}>
 			{/** Owner details */}
-			<Stack gap={1.5} alignItems="flex-start">
-				<HeadingComponent>{formSteps[0].label}</HeadingComponent>
+			<Stack gap={1.5}>
+				<H2>{formSteps[0].label}</H2>
```

Comment:
Would these by `h3`?
```
1: Review and submit
  2: Business details
    3: Owner details
```

Reply:
We don't have parent categories on this page (like we do on the view page).

The headings are this:

```
-- H2: We are planning a maintenance outage to upgrade the service on 25 May 2024 from 12pm to 5pm AEDT
-- H2: Progress
- H1: Apply for a Market stall permit form, Review and submit (aria label)
-- H2: Owner details
-- H2: Business details
-- H2: Business address
-- H2: Vehicle registration
-- H2: Trading time
-- H2: Food served
-- H2: Employees
-- H2: Food safety supervisor
-- H2: TODO - Upload documents
-- H2: Declaration
```